### PR TITLE
Add a proposal about Node-Group-Management

### PR DESCRIPTION
**Pull request by lonelyCZ**
Signed-off-by: lonelyCZ <531187475@qq.com>

**What type of PR is this?**
/kind design
/kind documentation

**What this PR does / why we need it**:
Add a proposal about Node-Group-Management.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fisherxu** [comment]
cc @vincentgoat

File: docs/proposals/node-group-management.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+	// +optional
+	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+}
+
+// NodeGroupStatus defines the observed state of NodeGroup
+type NodeGroupStatus struct {
+	// ContainedNodes represents names of all nodes the nodegroup contains.
+	// +optional
+	ContainedNodes []string `json:"containedNodes,omitempty"`
+}
```
Comment (fisherxu) [inline docs/proposals/node-group-management.md:124]: The nodegroup defination is still needed, let's use it :) @lonelyCZ @Congrool @vincentgoat

Comment (Congrool) [reply docs/proposals/node-group-management.md:124]: I'll update this proposal with the new design and put the `EdgeApplication` and `NodeGroup` API in it.

**Comment by lonelyCZ** [comment]
The proposal has been updated.

cc @fisherxu @Congrool @vincentgoat

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**